documentation/JavaFiddle_Requirements.txt

```diff
@@ -58,5 +58,8 @@ Reviewboard functionality
 - add/edit/read comments
 - approve/reject
 
+[Tsanda Artyom] Ideas:
+- add possibility to import project from an existing one on the local machine
```

Comment:
What if we just will look at a project file of IDE, where, if I am right, the whole information of project must be contained. Then, accroding to it, parse project. Finally, the main issue will be in understanding how project file works exactly.